ntt/parameters/baby_bear.h

```diff
@@ -14,65 +14,66 @@ const int S = 27;
 const fr_t forward_roots_of_unity[S + 1] = {
```

Comment:
Please add comments that this is modified to standard form as in plonky3, since other files might still be in mont form

Reply:
no it is still in Montgomery form, Plonky3 has in standard form.  Even before that change sppark had roots in Mont, but with different generator thats why I needed to change, I will add comments in sppark file that roots are modified to match with Plonky3 and they are not the same as in original

Reply:
I see. I updated the PR desc, since I was confused.

Just add comment to say "values were modified to match plonky3" is fine